lib/rig/distributed_set.ex

```diff
@@ -258,10 +258,11 @@ defmodule Rig.DistributedSet do
 
     n_deleted = :ets.select_delete(ets_table, match_spec)
 
-    if n_deleted > 0 do
-      Logger.debug(fn -> "Removed #{n_deleted} expired records" end)
-      RigMetrics.ControlInstrumenter.delete_blacklisted_session(n_deleted)
```

Comment:
Isn't this used in Prometheus?

Reply:
not sure... it didn't work that way. Looking at the [instrumenter module](https://github.com/Accenture/reactive-interaction-gateway/blob/master/lib/rig_metrics/control_instrumenter.ex), it seemed like this was still work-in-progress, so I've gathered it's better to remove it and add the feature later for real. @jakobwgnr what do you think?

Reply:
should be addressed in #157